provider/provider.go

```diff
@@ -611,7 +611,7 @@ func templateCondition(condition []byte, ctx map[string]engine.ChainTemplate) ([
 	s := strings.ReplaceAll(string(condition), `'{{`, "{{")
 	s = strings.ReplaceAll(s, `}}'`, "}}")
 
-	s, err := mustache.Render(s, true, ctx)
+	s, err := mustache.RenderRaw(s, true, ctx)
```

Comment:
Why render raw vs render just for my knowledge :)

Reply:
We were calling the `Render` method passing `true` as a second parameter, but the signature of Render doesn´t have a bool parameter. `RenderRaw` has, and I thought there was some reason in the past to use `RenderRaw` but the invocation was done mistakenly by calling `Render` instead...

Reply:
Thanks! that makes sense